test/test_stream.py

```diff
@@ -131,21 +131,29 @@ def test_high():
     alns = [a for a in stream.parse(gamfile, vg_pb2.Alignment)]
     assert len(alns) == gamfile_nof_alns
     # Rewrite it into a new file in two groups of 6 objects.
-    stream.dump(rw1_gamfile, *alns, buffer_size=len(alns)//2)
+    stream.dump(rw1_gamfile, *alns, buffer_size=len(alns)//2, gzip=gzip)
     # Read the rewritten file.
-    re_alns = [a for a in read_alns2(rw1_gamfile)]
+    re_alns = [a for a in read_alns2(rw1_gamfile, gzip=gzip)]
     # Check the length of the objects storing in both files.
     assert len(alns) == len(re_alns)
     # Rewrite again the read data.
-    write_objs2(rw2_gamfile, *re_alns)
+    write_objs2(rw2_gamfile, *re_alns, gzip=gzip)
     # Check whether the two generated files have the same the content.
-    assert compare(rw1_gamfile, rw2_gamfile)
+    assert compare(rw1_gamfile, rw2_gamfile, gzip=gzip)
     # Remove the generated files.
     os.remove(rw1_gamfile)
     os.remove(rw2_gamfile)
 
 
-def compare(first, second):
+def test_low_no_gzip():
```

Comment:
Is this function called anywhere in the test module?

Reply:
yes, it's found by `nosetest` (the tool you invoke in your makefile :-) )

Reply:
Oh, right. I forgot that.